doc/system_requirements.md

```diff
@@ -359,14 +359,24 @@ Obviously the detailed design is missing at the point when Kim runs the trace. T
 #### Include Only Artifact Types
 `req~include-only-artifact-types~1`
 
-OFT give users the option to include only a configurable set of artifact types during processing.
+OFT gives users the option to include only a configurable set of artifact types during processing.
 
 Covers:
 
   * [feat~requirement-tracing~1](#requirement-tracing)
 
 Needs: dsn
 
+#### Include Only Tags
+`req~include-only-tags~1`
+
+OFT gives users the option to include only a specification items contained in configurable set of tags during processing.
```

Comment:
* "include only **a** specification items contained" -> "include only specification items contained"
* "during processing": too generic. Either delete it or say "during import"